packages/gestalt-design-tokens/tokens/vr-theme/comp/spinner.json

```diff
@@ -0,0 +1,34 @@
+{
+  "comp": {
```

Comment:
can we moved the colors ones to comp-web-color-light.json?


 
![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/330f3323-22f5-4395-9fc4-4aec91c17283)

Reply:
I kept the color tokens in the VR theme as component tokens. Then mapped to them in comp-web-color-light.json in VR mapping theme

Reply:
the structure should be vr-theme/comp/spinner/default.json